SwrveConversationSDK/Conversation/SwrveConversationItemViewController.m

```diff
@@ -63,6 +64,86 @@ - (void)viewDidDisappear:(BOOL)animated {
     }
 }
 
+- (void)dealloc {
```

Comment:
Rather than dealloc I would move them to viewWillAppear and remove them in viewWillDisappear. If the object is not used but not dealloc'ed the same problems we had will happen with the view controller.